### PR TITLE
Server doc improvements

### PR DESCRIPTION
**Pull request by NachoToast**
image

### CODE REVIEW

File: docs/server/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    - `401` (unauthorized) if details are invalid on login.
+    - `400` (bad request) if details are invalid on signup.
+
+## Socket Connection System
+
+Users _must_ be given a JWT (made out of their username) before joining a lobby (or entering server hub chat), this is done like so:
+
+1. Check if JWT in local storage, if it is then skip all remaining steps.
+2. On lobby enter, ask to **give username**, **log in**, or **sign up** before actually attempting to join lobby.
+3. If **give username** is chosen, make a POST request to `/quicksignup`
```
Comment (NachoToast) [inline docs/server/README.md:57]: this is tbc

**Comment by PolarWolf314** [comment]
pretty based

- - ahyeee 😎ສ

**Review by PolarWolf314 — COMMENTED** [review]
bass

File: docs/server/README.md (hunk 1 above)
Comment (PolarWolf314) [reply docs/server/README.md:57]:
```py
print("hello world based department")
```

**Pull request merged**